src/mixxxmainwindow.cpp

```diff
@@ -169,6 +172,7 @@ void MixxxMainWindow::initializeQOpenGL() {
 #endif
 
 void MixxxMainWindow::initialize() {
+    qWarning() << "     $ initialize";
```

Comment:
Debug leftover? There are more warning that should be removed or become debug messages if it adds any value to the mixxx.log.

Reply:
This is the last commit with debug output which should help debugging in case something doesn't work as expected.
I'll remove that in case the other commits are considered okay.